### PR TITLE
added SQL query to reviews routes

### PR DESCRIPTION
**Pull request by MarieBaxter**

### CODE REVIEW

File: routes/reviewsRoutes.js [hunk 1]
```diff
@@ -3,12 +3,21 @@ const router = Router()
...
-  res.json([
-    { id: 1, rating: 3 },
-    { id: 2, rating: 4 },
-    { id: 3, rating: 5 }
-  ])
+router.get('/reviews', async (req, res) => {
+  try {
+    let house = ''
+    if (req.query.house) {
+      house = `WHERE house_id = ${req.query.house}`
```
Comment (ynotrusso) [inline routes/reviewsRoutes.js:10]: Once you've built this query… where are you using it then? It's not in use anywhere, this route always returns all reviews, not just the ones belonging to 1 house.

File: routes/reviewsRoutes.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ router.get('/reviews', async (req, res) => {
     const { rows } = await db.query(
       `SELECT * FROM reviews ${house} ORDER BY review_date DESC`
     )
+    const returnObject =
+      rows.length > 0 ? rows[0] : { error: 'house does not exist' }
```
Comment (ynotrusso) [inline routes/reviewsRoutes.js:16]: Why only return the first review? This is the `/reviews` route, which means it should return an array of reviews, whether it's 0, 1 or many reviews.

**Review by ynotrusso — CHANGES_REQUESTED** [review]
Please see comments inline.

File: routes/reviewsRoutes.js (hunk 2 above)
Comment (MarieBaxter) [reply routes/reviewsRoutes.js:16]: Ok I fixed this so it returns an array with all the reviews for the house :)

File: routes/reviewsRoutes.js (hunk 1 above)
Comment (MarieBaxter) [reply routes/reviewsRoutes.js:10]: Sorry I don't fully understand your question!

The instructions say you should be able to filter by reviews/?house=x

I tested the code and if you just put /reviews it returns all of the reviews, if you put reviews/?house=x it returns the reviews for the specified house, and if the house_id doesn't exist it return 'house does not exist'.

What am I missing?

**Review by ynotrusso — APPROVED** [review]

**Pull request merged**